packages/generators/app/src/utils/git.ts

```diff
@@ -27,8 +27,15 @@ export default async function tryGitInit(rootDir: string) {
 
     await execa('git', ['init'], { stdio: 'ignore', cwd: rootDir });
 
+    await execa('git', ['add', '.'], { stdio: 'ignore', cwd: rootDir });
+    await execa('git', ['commit', '-m', 'Initial commit from Strapi'], {
```

Comment:
What if the users' git server expect signed commits or other configs? (we used to required that on Github, hence the question).

Having to rewrite the commits history would create some friction, wouldn't it? Or do we expect this to be a niche scenario, thus only supporting the main usecase by default?

Reply:
we were doing it for the starters and it's done in most tools nowdays I think. no strong opinions honestly

Reply:
Yep, after thinking a bit about this, I don't think it's worth taking this specific usecase into consideration. It's a nice to have that even github does 🤷 

Let's keep it